appendices/versions.txt

```diff
@@ -188,37 +188,70 @@ number of key extensions into the core API:
 
 include::{generated}/meta/promoted_extensions_VK_VERSION_1_1.txt[]
 
-The only changes to the functionality added by these extensions were to
-`VK_KHR_shader_draw_parameters`, which had the
-<<features-shaderDrawParameters, pname:shaderDrawParameters>> feature bit
-added to determine support in the core API, and
-<<features-variablePointersStorageBuffer,
-pname:variablePointersStorageBuffer>> from `VK_KHR_variable_pointers` was
-made optional.
+All differences in behavior between these extensions and the corresponding
+Vulkan 1.1 functionality are summarized below.
 
-[[versions-1.1-new-features]]
-Additionally, Vulkan 1.1 added support for <<shaders-group-operations, group
-operations>> with <<shaders-scope-subgroup, subgroup scope>>,
-<<memory-protected-memory, protected memory>>, and a new command to
-<<vkEnumerateInstanceVersion, enumerate the instance version>>.
+=== Differences relative to `VK_KHR_16bit_storage`
+
+If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
+the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
+is optional.
+Support for this feature is defined by
+slink:VkPhysicalDevice16BitStorageFeatures::pname:storageBuffer16BitAccess
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_KHR_sampler_ycbcr_conversion`
 
 If the `<<VK_KHR_sampler_ycbcr_conversion>>` extension is not supported,
 support for the <<features-samplerYcbcrConversion,samplerYcbcrConversion>>
 feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceSamplerYcbcrConversionFeatures::pname:samplerYcbcrConversion
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
-=== Differences relative to `VK_KHR_16bit_storage`
+=== Differences relative to `VK_KHR_shader_draw_parameters`
```

Comment:
So I actually don't think there should be a `Differences relative to VK_KHR_shader_draw_parameters` (and why we didn't add it to start with) because this suggest there WAS a `shaderDrawParameters` prior to 1.1

The reason for things like VK_KHR_16bit_storage is because 
- in 1.0 if you exposed the `storageBuffer16BitAccess` is guaranteed
- in 1.1 the extension is not core, but a 1.1 implementation doesn't guarantee `storageBuffer16BitAccess`

This logic is not an issue with `VK_KHR_shader_draw_parameters` as it was all additive

Reply:
Same thing with`VK_KHR_variable_pointers`/`VkPhysicalDeviceVariablePointersFeatures`

Reply:
I used the 1.2 chapter as a guide. See e.g. [Differences relative to VK_KHR_sampler_mirror_clamp_to_edge](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#_differences_relative_to_vk_khr_sampler_mirror_clamp_to_edge). Basically anything that is not trivial promotion has this difference section.

Reply:
That is not the criteria for this section

Anything non-trival (or any changes) belongs in the Extension appendix (ex `./chapters/VK_KHR_sampler_mirror_clamp_to_edge.txt`)

 This section is ONLY for things that were non-optional that became optional in order to fold the extension into core (

Example: `VK_KHR_timeline_semaphore` the `timelineSemaphore` was added in `VkPhysicalDeviceVulkan12Features` but there is not `Difference relative to` section because a 1.2 implementation has to support `timelineSemaphore`

The sole purpose of these sections was to help point out "These extensions are going into core, but the original guarantees of features still need to be queried as the 'guaranteed feature support' wasn't folded into core"

Reply:
Timeline semaphores were trivial promotion.

For 1:1 comparison see `VK_KHR_draw_indirect_count` or the `VK_KHR_sampler_mirror_clamp_to_edge`.

Previously the feature was non-optional (as determined via the feature alias of the extension support). Now it is optional in core.

Reply:
I am getting 2nd opinions inside Khronos, wanted to leave comment to acknowledge I haven't forgotten about this :)

Reply:
So after thinking about it more, I agree with these changes as whole